code/controllers/subsystem/monitor.dm

```diff
@@ -48,13 +50,18 @@ SUBSYSTEM_DEF(monitor)
 	. = ..()
 	RegisterSignal(SSdcs, COMSIG_GLOB_OPEN_TIMED_SHUTTERS_LATE, .proc/set_groundside_calculation)
 	RegisterSignal(SSdcs, COMSIG_GLOB_DROPSHIP_HIJACKED, .proc/set_shipside_calculation)
+	balance_system_authorised = CONFIG_GET(number/balance_system_authorised)
 
 /datum/controller/subsystem/monitor/fire(resumed = 0)
 	current_points = calculate_state_points() / max(GLOB.alive_human_list.len + GLOB.alive_xeno_list.len, 10)//having less than 10 players gives bad results
 	if(gamestate == GROUNDSIDE)
 		process_human_positions()
 		FOB_hugging_check()
 	set_state(current_points)
+	var/proposed_balance_buff = balance_xeno_team()
+	if(abs(proposed_balance_buff - GLOB.xeno_stat_multiplicator_buff) >= 0.05)
```

Comment:
why the absolute value?

Reply:
because you don't know if proposed_balance_buff is greater or not than the current xeno buff? 
The system can buff xeno, then later burrowed are finallly took so the system stop buffing xeno